daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLIParsing.scala

```diff
@@ -358,7 +358,7 @@ class TestCLIParsing {
 
     runCLI(args"parse -s $schema -r unknown") { cli =>
       cli.sendLine("12", inputDone = true)
-      cli.expectErr("No root element found for unknown in any available namespace")
+      cli.expectErr("No root element found for {}unknown")
```

Comment:
This is potentially confusing, and maybe not correct? `unknown` and `{}unknown` are technically different things. The former means "some element called unknown in any namespace". But the later means "some element called unknown explicitly in no-namespace". The error message should probably not include the curly braces here.